src/main/java/org/zalando/nakadi/metrics/StreamKpiData.java

```diff
@@ -0,0 +1,24 @@
+package org.zalando.nakadi.metrics;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public class StreamKpiData {
+        private AtomicLong bytesSent = new AtomicLong(0);
+        private AtomicLong numberOfEventsSent = new AtomicLong(0);
```

Comment:
Why do you use atomics here? It seems that these values are accessed/set within one thread only.

Reply:
Yes, I went home yesterday and realised it's completely useless. I'll change it to usual long.

Reply:
https://github.com/zalando/nakadi/pull/813/commits/c946a4403f66393729112d2d7f0edcca8a423224